src/jobs/check-health.yml

```diff
@@ -0,0 +1,48 @@
+description: >
+  This job checks the specified AWS region for any issues. The job passes if there are no issues found.
+  If there are issues in the region, the job will continue to check for issues every 10 seconds until
+  no issues are detected. If the max attempts have been reached, the job will fail.
+
+executor: <<parameters.executor>>
+
+parameters:
+  executor:
+    type: executor
+  aws-region-to-check:
+    type: env_var_name
+    default: AWS_REGION
+    description: >
+      The AWS region to be checked for issues. It defaults to AWS_REGION that's set as an environment variable
+  max-poll-attempts:
+    type: integer
+    default: 6
+    description: >
+      The max number of attempts to recheck for issues in the specified AWS region. Each attempt takes
+      10 seconds. By default, 6 attempts will be made for a total of 1 minute.
+  aws-access-key-id:
+    type: env_var_name
+    description: aws access key id override
+    default: AWS_ACCESS_KEY_ID
```

Comment:
technically nothing wrong with the environment variable, but this could potentially be used by someone else? better to make it PARAM_AWS? applies to the rest of the environment variables

Reply:
The documentation for AWS uses the same environment varaibles for their keys and regions. Good point on how it can be used by someone else. In this case though, that's exactly what we want. Most customers who use AWS orbs use more than one and their usage of enviroment variables overlap so it's good to have uniformity.